NoSuchMethodError: 'java.util.List org.eclipse.aether.spi.connector.layout.RepositoryLayout.getChecksumAlgorithmFactories()'

After releasing 1.0.20, starting with an empty .m2 repository produces the following exception:
```
Failed to execute goal com.gluonhq:gluonfx-maven-plugin:1.0.20:compile (default-cli) on project hellogluon: Execution default-cli of goal com.gluonhq:gluonfx-maven-plugin:1.0.20:compile failed: An API incompatibility was encountered while executing com.gluonhq:gluonfx-maven-plugin:1.0.20:compile: java.lang.NoSuchMethodError: 'java.util.List org.eclipse.aether.spi.connector.layout.RepositoryLayout.getChecksumAlgorithmFactories()'

...
Caused by: java.lang.NoSuchMethodError: 'java.util.List org.eclipse.aether.spi.connector.layout.RepositoryLayout.getChecksumAlgorithmFactories()'
    at org.eclipse.aether.connector.basic.BasicRepositoryConnector.get (BasicRepositoryConnector.java:189)
    at org.eclipse.aether.internal.impl.DefaultArtifactResolver.performDownloads (DefaultArtifactResolver.java:514)
    at org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolve (DefaultArtifactResolver.java:402)
    at org.eclipse.aether.internal.impl.DefaultArtifactResolver.resolveArtifacts (DefaultArtifactResolver.java:229)
    at org.eclipse.aether.internal.impl.DefaultRepositorySystem.resolveArtifacts (DefaultRepositorySystem.java:270)
    at com.gluonhq.utils.MavenArtifactResolver.resolve (MavenArtifactResolver.java:201)
    at com.gluonhq.utils.MavenArtifactResolver.resolve (MavenArtifactResolver.java:182)
    at com.gluonhq.attach.AttachArtifactResolver.lambda$findArtifactsForTarget$2 (AttachArtifactResolver.java:72)

```

--- a/pom.xml
+++ b/pom.xml
@@ -47,7 +47,7 @@
     <source.plugin.version>3.3.0</source.plugin.version>
     <javadoc.plugin.version>3.5.0</javadoc.plugin.version>
     <gpg.plugin.version>3.1.0</gpg.plugin.version>
-    <maven.resolver.version>1.9.15</maven.resolver.version>
+    <maven.resolver.version>1.7.3</maven.resolver.version>
     <substrate.version>0.0.61-SNAPSHOT</substrate.version>
   </properties>
 
@@ -60,7 +60,7 @@
     <dependency>
       <groupId>org.apache.maven</groupId>
       <artifactId>maven-plugin-api</artifactId>
-      <version>3.9.4</version>
+      <version>3.8.8</version>
       <scope>provided</scope>
     </dependency>
     <dependency>
@@ -82,7 +82,7 @@
     <dependency>
       <groupId>org.apache.maven</groupId>
       <artifactId>maven-core</artifactId>
-      <version>3.9.4</version>
+      <version>3.8.8</version>
       <scope>provided</scope>
     </dependency>
     <dependency>

